server/docker-base/Dockerfile

```diff
@@ -581,6 +435,82 @@ COPY template.build.gradle ${EXTENSION_BUILD_GRADLE_TEMPLATE}
 COPY template.gradle.properties ${EXTENSION_GRADLE_PROPERTIES_TEMPLATE}
 COPY template.local.properties ${EXTENSION_LOCAL_PROPERTIES_TEMPLATE}
 
+#
```

Comment:
I moved it last since rebuilding gets a lot faster when I constantly change variables.